pkg/fakes/mock_blockchain.go

```diff
@@ -109,6 +115,28 @@ func (blockChain *MockBlockChain) LastBlock() (*big.Int, error) {
 	return blockChain.lastBlock, nil
 }
 
+func (blockChain *MockBlockChain) GetStorageAt(account common.Address, key common.Hash, blockNumber *big.Int) ([]byte, error) {
+	blockChain.GetStorageAtPassedAccounts = append(blockChain.GetStorageAtPassedAccounts, account)
+	blockChain.GetStorageAtPassedKeys = append(blockChain.GetStorageAtPassedKeys, key)
+	blockChain.GetStorageAtPassedBlockNumber = blockNumber
+
+	if len(blockChain.storageValuesToReturn) > 0 {
+		storageToReturn := blockChain.storageValuesToReturn[account]
+
+		return storageToReturn, blockChain.GetStorageAtError
+	} else {
+		return nil, blockChain.GetStorageAtError
+	}
```

Comment:
Do we need this `if`/`else`? I would have thought that line 126 would return the same thing as line 128 if there was no entry for the account, but could be wrong

Reply:
yeah, nice catch! that was some cruft from refactoring the mock GetStorageAt method. ✂️